src/test/java/com/karankumar/booksapi/repository/BookRepositoryTest.java

```diff
@@ -105,8 +107,47 @@ private Book createBookwithISBN() {
         book.setIsbn13(ISBN);
         return book;
     }
+  @Test
+    @DisplayName("find book by title")
+    void findBookByTitle() {
+        // given
+        createAndSaveAuthors();
+        Book book = createBookWithTitle();
+        bookRepository.save(book);
+
+        // when
+        Book result = bookRepository.findBookByTitle(TITLE);
+
+        // then
+        assertThat(result).isEqualTo(book);
+    }
+
+    @Test
+    @DisplayName("find book by title case insensitive")
+    void findBookByTitleCaseInsensitive() {
+        // given
+        createAndSaveAuthors();
+        Book book = createBookWithTitle();
+        bookRepository.save(book);
 
+        // when
+        Book result = bookRepository.findBookByTitle(TITLE.toLowerCase());
+
+        // then
+        assertThat(result).isEqualTo(book);
+    }
```

Comment:
As per our style guide, could you please have one blank line between the end of one method and the start of the next?